mate-panel/button-widget.c

```diff
@@ -57,876 +57,873 @@ G_DEFINE_TYPE_WITH_PRIVATE (ButtonWidget, button_widget, GTK_TYPE_BUTTON)
+                                        cairo_image_surface_get_height (surface));
 
-	do_colorshift (new, surface, 30);
+    do_colorshift (new, surface, 30);
 
-	cr = cairo_create (new);
-	cairo_set_operator (cr, CAIRO_OPERATOR_DEST_IN);
-	cairo_mask_surface (cr, surface, 0, 0);
-	cairo_destroy (cr);
+    cr = cairo_create (new);
+    cairo_set_operator (cr, CAIRO_OPERATOR_DEST_IN);
+    cairo_mask_surface (cr, surface, 0, 0);
+    cairo_destroy (cr);
 
-	return new;
+    return new;
 }
 
 static void
 button_widget_realize(GtkWidget *widget)
 {
-	gtk_widget_add_events (widget, GDK_POINTER_MOTION_MASK |
-			       GDK_POINTER_MOTION_HINT_MASK |
-			       GDK_KEY_PRESS_MASK);
+    gtk_widget_add_events (widget, GDK_POINTER_MOTION_MASK |
+                           GDK_POINTER_MOTION_HINT_MASK |
+                           GDK_KEY_PRESS_MASK);
 
-	GTK_WIDGET_CLASS (button_widget_parent_class)->realize (widget);
+    GTK_WIDGET_CLASS (button_widget_parent_class)->realize (widget);
 
-	BUTTON_WIDGET (widget)->priv->icon_theme = gtk_icon_theme_get_for_screen (gtk_widget_get_screen (widget));
-	g_signal_connect_object (BUTTON_WIDGET (widget)->priv->icon_theme,
-				 "changed",
-				 G_CALLBACK (button_widget_icon_theme_changed),
-				 widget,
-				 G_CONNECT_SWAPPED);
+    BUTTON_WIDGET (widget)->priv->icon_theme = gtk_icon_theme_get_for_screen (gtk_widget_get_screen (widget));
+    g_signal_connect_object (BUTTON_WIDGET (widget)->priv->icon_theme,
+                             "changed",
+                             G_CALLBACK (button_widget_icon_theme_changed),
+                             widget,
+                             G_CONNECT_SWAPPED);
 
-	button_widget_reload_surface (BUTTON_WIDGET (widget));
+    button_widget_reload_surface (BUTTON_WIDGET (widget));
 }
 
 static void
 button_widget_unrealize (GtkWidget *widget)
 {
-	g_signal_handlers_disconnect_by_func (BUTTON_WIDGET (widget)->priv->icon_theme,
-					      G_CALLBACK (button_widget_icon_theme_changed),
-					      widget);
+    g_signal_handlers_disconnect_by_func (BUTTON_WIDGET (widget)->priv->icon_theme,
+                                          G_CALLBACK (button_widget_icon_theme_changed),
+                                          widget);
 
-	GTK_WIDGET_CLASS (button_widget_parent_class)->unrealize (widget);
+    GTK_WIDGET_CLASS (button_widget_parent_class)->unrealize (widget);
 }
 
 static void
 button_widget_unset_surfaces (ButtonWidget *button)
 {
-	if (button->priv->surface)
-		cairo_surface_destroy (button->priv->surface);
-	button->priv->surface = NULL;
+    if (button->priv->surface)
+        cairo_surface_destroy (button->priv->surface);
 
-	if (button->priv->surface_hc)
-		cairo_surface_destroy (button->priv->surface_hc);
-	button->priv->surface_hc = NULL;
+    button->priv->surface = NULL;
+
+    if (button->priv->surface_hc)
+        cairo_surface_destroy (button->priv->surface_hc);
+
+    button->priv->surface_hc = NULL;
```

Comment:
@mate-desktop/core-team 
Not sure about adding a blank line after the if clauses.
`button->priv->surface = NULL;`
and
`button->priv->surface_hc = NULL;`
doesn't belongs to the if clauses?

Reply:
good than adding a blank line is OK.

Reply:
If indentation is correct either way is fine, but I don't think a blank line is a bad idea, so I'm happy with it.